### DISCUSSION TITLE
TCPStream.h missing when compiling for raspberry pi

### DISCUSSION THREAD

**Discussion by jrdarrah** [terjeio/grblHAL#194] [Q&A]
Thanks for getting me past the LWIP error.  I have the next error but it  makes no sense.  It is able to find and start compiling TCPStream.c which has the TCPStream.h included.  I can see both files in the Networking directory so I don't understand why I get the error below.  To make sure I have the directories properly aligned. Starting from

Arduino
    libraries
       grblHAL_Teensy4
           examples
           src
               grbl
                networking

/home/pi/arduino-1.8.13/hardware/teensy/../tools/arm/bin/arm-none-eabi-g++ -E -CC -x c++ -w -g -Wall -ffunction-sections -fdata-sections -nostdlib -std=gnu++14 -fno-exceptions -fpermissive -fno-rtti -fno-threadsafe-statics -felide-constructors -Wno-error=narrowing -mthumb -mcpu=cortex-m7 -mfloat-abi=hard -mfpu=fpv5-d16 -D__IMXRT1062__ -DTEENSYDUINO=153 -DARDUINO=10813 -DARDUINO_TEENSY41 -DF_CPU=600000000 -DUSB_SERIAL -DLAYOUT_US_ENGLISH -I/home/pi/arduino-1.8.13/hardware/teensy/avr/cores/teensy4 -I/home/pi/Arduino/libraries/grblHAL_Teensy4/src -I/home/pi/Arduino/libraries/teensy41_ethernet-master/src /home/pi/Arduino/libraries/grblHAL_Teensy4/src/networking/TCPStream.c -o /dev/null -DARDUINO_LIB_DISCOVERY_PHASE
Alternatives for TCPSTream.h: []
ResolveLibrary(TCPSTream.h)
  -> candidates: []
In file included from /home/pi/Arduino/libraries/grblHAL_Teensy4/src/networking/TCPStream.c:40:0:
/home/pi/Arduino/libraries/grblHAL_Teensy4/src/networking/networking.h:30:23: fatal error: TCPSTream.h: No such file or directory
compilation terminated.
Using library grblHAL_Teensy4 at version 1.1.7 in folder: /home/pi/Arduino/libraries/grblHAL_Teensy4 
Using library teensy41_ethernet-master at version 2.0.2 in folder: /home/pi/Arduino/libraries/teensy41_ethernet-master 
Error compiling for board Teensy 4.1.

**Comment by terjeio** [comment]
There are subtle spelling mistakes (or rather case mismatches) that is masked by Windows (it is not case sensitive vs. filenames), I see now that a pull request to fix that was against the test branch. In _networking.h_ there are two errors:
```
#if TELNET_ENABLE
#include "TCPSTream.h"
#endif

#if WEBSOCKET_ENABLE
#include "WsSTream.h"
#endif

```
should be:
```
#if TELNET_ENABLE
#include "TCPStream.h"
#endif

#if WEBSOCKET_ENABLE
#include "WsStream.h"
#endif
```
